rpc/grpc/client/block_results_service.go

```diff
@@ -0,0 +1,64 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	abci "github.com/cometbft/cometbft/abci/types"
+	cmtproto "github.com/cometbft/cometbft/proto/tendermint/types"
+
+	"github.com/cosmos/gogoproto/grpc"
+
+	brs "github.com/cometbft/cometbft/proto/tendermint/services/block_results/v1"
+)
+
+type ResultBlockResults struct {
+	Height                int64                     `json:"height"`
+	TxsResults            []*abci.ExecTxResult      `json:"txs_results"`
+	FinalizeBlockEvents   []*abci.Event             `json:"finalize_block_events"`
+	ValidatorUpdates      []*abci.ValidatorUpdate   `json:"validator_updates"`
+	ConsensusParamUpdates *cmtproto.ConsensusParams `json:"consensus_param_updates"`
+	AppHash               []byte                    `json:"app_hash"`
+}
+
+// BlockResultsServiceClient provides the block results of a given height (or latest if none provided).
+type BlockResultsServiceClient interface {
+	GetBlockResults(ctx context.Context, req brs.GetBlockResultsRequest) (*ResultBlockResults, error)
```

Comment:
This should just be:
```suggestion
	GetBlockResults(ctx context.Context, height int64) (*ResultBlockResults, error)
```